tunslip6.c

```diff
@@ -545,15 +650,36 @@ stty_telos(int fd)
 }
 
 int
+devopen_readlink(const char *dev, int flags)
```

Comment:
This is just so that we always try to readlink because when a new /tmp/slip.dev is created it will point to a new place.